tests/tests/swfs/avm2/xml_basic/Test.as

```diff
@@ -1,5 +1,6 @@
 ﻿package {
 	public class Test {
+		import flash.utils.*;
```

Comment:
I think this line is unnecessary.

Reply:
Yeah, you're right. Remnants from some other testing I was doing at the same time